tests/h/search/index_test.py

```diff
@@ -94,6 +102,111 @@ def test_you_can_filter_annotations_by_creation_time(self, factories, index, sea
         response = search.filter("range", created={"gte": before}).execute()
         assert SearchResponseWithIDs([annotation.id]) == response
 
+    def test_you_can_filter_annotations_by_updated_time(self, factories, index, search):
+        update_time = datetime.datetime.now()
+        annotation = factories.Annotation.build(id="test_annotation_id", updated=update_time)
+
+        index(annotation)
+
+        response = search.filter("range", updated={"gte": update_time}).execute()
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_id(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_ann_id")
+
+        index(annotation)
+
+        response = search.filter("term", id="test_ann_id").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_user(self, factories, index, search):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", user="acct:someone@example.com").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_user_raw(self, factories, index, search):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", user_raw="acct:someone@example.com").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_tags(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_annotation_id", tags=["tag1", "tag2"])
+
+        index(annotation)
+
+        response1 = search.filter("term", tags=["tag1"]).execute()
+        response2 = search.filter("term", tags=["tag2"]).execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response1
+        assert SearchResponseWithIDs([annotation.id]) == response2
```

Comment:
I'm wondering what is the difference between `"tags"` and `"tags_raw"`? Looking in `config.py` it looks like it's that `"tags"` uses `uni_normalizer`: https://github.com/hypothesis/h/blob/61fe02289488e0bf88d54aa37dae51a8300c00a5/h/search/config.py#L31-L32

This must enable some filtering functionality for `"tags"` that isn't enabled for `"tags_raw"`. Can we figure out what that is, and write a test for it? Can be a separate pr.

Reply:
It looks like we are using the analysed `tags`field for searching and the `tags_raw` field in TagsAggregation in query.py